clangd/ClangdLSPServer.cpp

```diff
@@ -65,10 +66,18 @@ void ClangdLSPServer::onInitialize(Ctx C, InitializeParams &Params) {
             {"referencesProvider", true},
             {"workspaceSymbolProvider", true},
         }}}});
+  std::vector<Path> ExclusionList;
+
+  if (Params.initializationOptions.hasValue()) {
+    for (auto CurrentPath : Params.initializationOptions.getValue()) {
+      ExclusionList.push_back(CurrentPath);
+    }
+  }
+
   if (Params.rootUri && !Params.rootUri->file.empty())
-    Server.setRootPath(Params.rootUri->file);
+    Server.setRootPath(Params.rootUri->file, ExclusionList);
```

Comment:
change to set setConfiguration and pass both rootUri and Params.initializationOptions?